scripts/linux/config-etc-hosts.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+
+SCRIPT_NAME="manage-hosts.sh"
+SCRIPT_PATH="/vagrant/scripts/linux"
+
+PLAYGROUND_NAME="$1"
+#PLAYGROUND_NAME="k8s-p9"
+
+MASTER_1="k8s-master-1"
+MINION_1="k8s-minion-1"
+MINION_2="k8s-minion-2"
+MINION_3="k8s-minion-3"
+
+MASTER_1_IP="192.168.0.10"
+MINION_1_IP="192.168.0.30"
+MINION_2_IP="192.168.0.31"
+MINION_3_IP="192.168.0.32"
+
+
+$SCRIPT_PATH/$SCRIPT_NAME remove $MASTER_1.$PLAYGROUND_NAME.local
+$SCRIPT_PATH/$SCRIPT_NAME remove $MINION_1.$PLAYGROUND_NAME.local
+$SCRIPT_PATH/$SCRIPT_NAME remove $MINION_2.$PLAYGROUND_NAME.local
+$SCRIPT_PATH/$SCRIPT_NAME remove $MINION_3.$PLAYGROUND_NAME.local
+
+$SCRIPT_PATH/$SCRIPT_NAME add $MASTER_1.$PLAYGROUND_NAME.local $MASTER_1_IP
```

Comment:
I get what you're doing here, but I guess it's easier to manage with Ansible...

See https://docs.ansible.com/ansible/latest/modules/lineinfile_module.html#examples

Reply:
I can work on this bit

Reply:
ok, see my comment at the end